includes/wikia/services/usercommands/ShortcutsUserCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * Displays Global Shortcuts in toolbar
+ */
+class ShortcutsUserCommand extends UserCommand {
+    
+    protected $overflow = false;
+    protected $defaultRenderType = 'link';
+    
+    protected function buildData() {
+        $this->available = F::app()->wg->user->isLoggedIn();
```

Comment:
I think it's unnecessary to call it twice, better cache the result in a local variable.